packages/driver/test/cypress/fixtures/xhr-triggered.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>jQuery 3.2.1 Fixture</title>
+    <script type="text/javascript" src="/fixtures/jquery-3.2.1.js"></script>
+  </head>
+  <body>
+    <button id="trigger-xhr">Trigger XHR</button>
+    <script>
+      $('#trigger-xhr').click(function(event) {
+        $.post('/test-xhr')
```

Comment:
I would love to see an assertion here - what does the client receive for the object and for the `null` - does the client without a `null` stub receive ''?

Reply:
Added a callback that sets a div element with stringified data. 

And `cy` checks if the div has the correct string.